scmgr/README.md

```diff
@@ -31,7 +31,7 @@ you can use with scmgr. In the following examples, we suppose that the
 directory and that you installed `scmgr` using
 
 ```bash
-go get github.com/dedis/cothority/scmgr
+go build ../scmgr
```

Comment:
```suggestion
go build .
```

Reply:
no, because the previous `cd cothority/conode` supposes that the current `wd` is `cothority/conode`. This `wd` is also needed for the other commands.